apps/site/assets/js/support-form.js

```diff
@@ -356,16 +363,21 @@ function responseRequested($) {
 }
 
 function setupValidation($) {
-  ["#privacy", "#comments", "#email", "#first_name", "#last_name"].forEach(
-    selector => {
-      const $selector = $(selector);
-      $selector.on("keyup blur input change", () => {
-        if (validators[selector.slice(1)]($)) {
-          displaySuccess($, selector);
-        }
-      });
-    }
-  );
+  [
+    "#privacy",
+    "#comments",
+    "#email",
+    "#first_name",
+    "#last_name",
+    "#vehicle"
+  ].forEach(selector => {
```

Comment:
I don't remember whether we support IE < 9, but `forEach` isn't supported there: https://caniuse.com/mdn-javascript_builtins_array_foreach

Reply:
Thankfully we only support IE 11+!